main.go

```diff
@@ -115,25 +126,32 @@ func realMain(args []string) int {
 			sem <- 0
 			defer func() { <-sem }()
 
-			loader := gojsonschema.NewReferenceLoader(fileUri(path))
-			result, err := schema.Validate(loader)
-			switch {
-			case err != nil:
-				msg := fmt.Sprintf("%s: error: %s", path, err)
+
+			loader, err := jsonLoader(path)
+			if err != nil {
+				msg := fmt.Sprintf("%s: unable to load doc: %s\n", *schemaFlag, err)
```

Comment:
```suggestion
				msg := fmt.Sprintf("%s: error: %s\n", path, err)
```

This should have the same message format as `schema.Validate` errors since these are non-fatal document errors and something like `yajsv ... | grep error:` will catch them. I originally wrote this to validate ~3.2M JSON docs and being able to do that when it runs for potentially hours was super useful.

Reply:
Although now that file loading and schema validation are split it might be useful to add extra context to the errors, e.g.

```go
msg := fmt.Sprintf("%s: error: load doc: %s\n", path, err)
```

Reply:
yep. done